x-pack/plugins/observability_solution/apm/public/components/shared/entity_enablement/unauthorized_modal.tsx

```diff
@@ -54,6 +57,18 @@ export function Unauthorized({
               })}
             </EuiButton>
           }
+          cancelButtonText={
+            <EuiLink
+              target="_blank"
+              data-test-subj="apmUnauthorizedLinkExternal"
+              href="https://ela.st/new-experience-services"
+              external
+            >
+              {i18n.translate('xpack.apm.unauthorized.linkLinkLabel', {
+                defaultMessage: 'See how to enable EEM',
```

Comment:
Did we make it clear for users what `EEM` is?

Reply:
This link will redirect the user to the documentation regarding EEM
Here is the preview https://elastic-dot-co-docs-production-byr2bloo0-elastic-dev.vercel.app/current/serverless/observability/elastic-entity-model